lua/packer/plugin_types/git.lua

```diff
@@ -41,22 +41,32 @@ local function has_wildcard(tag)
   return string.match(tag, '*') ~= nil
 end
 
-local break_tag_pattern = [=[[bB][rR][eE][aA][kK]!?:]=]
+local break_tag_pattern = [=[[bB][rR][eE][aA][kK]:]=]
 local breaking_change_pattern = [=[[bB][rR][eE][aA][kK][iI][nN][gG][ _][cC][hH][aA][nN][gG][eE]]=]
+local type_exclam_pattern = [=[[a-zA-z]+!:]=]
+local type_scope_exclam_pattern = [=[[a-zA-z]+%([^)]+%)!:]=]
```

Comment:
Should be `[a-zA-Z]`.

Reply:
Yes, the `+` is not that necessary since we don't match the beginning of the string, or whitespaces before the type, or whatever....
Suggestion applied, thank you!